src/controller/python/chip/yaml/parser.py

```diff
@@ -29,11 +29,119 @@
 from chip.yaml.errors import ParsingError, UnexpectedParsingError
 from .data_model_lookup import *
 import chip.yaml.format_converter as Converter
+from .response_storage import ResponseStorage
 
 _SUCCESS_STATUS_CODE = "SUCCESS"
+_NODE_ID_DEFAULT = 0x12345
+_ENDPOINT_DETAULT = ''  # TODO why is this an empty string
+_CLUSTER_DEFAULT = ''
+_TIMEOUT_DEFAULT = 90
 logger = logging.getLogger('YamlParser')
 
 
+@dataclass
+class _CommonObjects:
```

Comment:
I don't like this `_CommonObjects` name. Arguably I am overloading this, and should remove this and instead provide all 3 arguments separately to each of the function arguments.